### PR TITLE
Ensembl workflow

### PR DESCRIPTION
**Pull request by anilthanki**

### CODE REVIEW

**Comment by nsoranzo** [comment]
The rest of the repository is under MIT license: https://github.com/TGAC/tgac-galaxytools/blob/master/LICENSE , are you sure we need GPL license for just the workflow?

File: workflow/GeneSeqToFamily/readme.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+GeneSeqToFamily: the Ensembl GeneTrees pipeline as a Galaxy workflow
+=================
+
+
+# Introduction 
+ GeneSeqToFamily, an open-source Galaxy workflow based on the Ensembl GeneTrees pipeline. The Ensembl GeneTrees pipeline [1] infers the evolutionary history of gene families, represented as gene trees. It is a computational pipeline that comprises  clustering, multiple alignment, and tree generation (using TreeBeST), to discover familial relationship.
```
Comment (nsoranzo) [inline workflow/GeneSeqToFamily/readme.md]: "GeneSeqToFamily, an open-source" => "GeneSeqToFamily is an open-source"

File: workflow/GeneSeqToFamily/readme.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+* CDS sequences
+* A species tree
+* Gene feature information in JSON format
+
+## Workflow
+
+The pipeline is made up of 7 main steps:
+* Translation from CDS to protein sequences
+* all-vs-all BLASTP of protein sequences
+* Cluster sequences using hcluster_sg
```
Comment (nsoranzo) [inline workflow/GeneSeqToFamily/readme.md]: and BLASTP scores

File: workflow/GeneSeqToFamily/readme.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+We have developed various tools to help with data preparation for the workflow. This includes tools for retrieving sequences, features and gene trees from Ensembl using its REST API, and tools to parse Ensembl results into the required formats for the workflow. We also developed a tool to merge gene feature files and convert them from GFF3 (Gene Feature File) to JSON format, which is then used to generate the Aequatus dataset.
+
+
+# Result
+Resulted gene families can be visualised using the Aequatus.js interactive tool, which is developed as part of the Aequatus software.
+
+The Aequatus.js plugin provides an interactive visual representation of the phylogenetic and structural relationships among the homologous genes, using a shared colour scheme for coding regions to represent homology in internal gene structure alongside their corresponding gene trees. It is also able to indicate insertions and deletions in homologous genes with respect to shared ancestors.
+
+# List of tools
+GeneSeqToFamily requires the following tools to run workflow successfully:
```
Comment (nsoranzo) [inline workflow/GeneSeqToFamily/readme.md]: "run workflow" => "run the workflow"

File: workflow/GeneSeqToFamily/readme.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+* TranSeq
+* filter fasta by ID
+* BLAST
+* BLAST parse
```
Comment (nsoranzo) [inline workflow/GeneSeqToFamily/readme.md]: s/parse/parser

File: workflow/GeneSeqToFamily/readme.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+* TranAlign
+* TreeBeST
+* Aequatus generator
+
+Some tools for data conversion during workflow:
+* cut
+* Fasta width
+* Fasta to tabular
+
+Helper tools for data preparation
```
Comment (nsoranzo) [inline workflow/GeneSeqToFamily/readme.md]: Add a ':' at the end

File: workflow/GeneSeqToFamily/readme.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+Ensembl REST API - This includes tools for retrieving sequences, features and gene trees from Ensembl using its REST API,
+Ensembl Parser - Tools to parse Ensembl results into the required format for workflow.
+gff3-to-json - We also developed tool to merge gene feature files and convert them from GFF3 (Gene Feature File) to JSON format.
+
+
+# Reference:
+* Vilella AJ, Severin J, Ureta-Vidal A, Heng L, Durbin R, Birney E: EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates. Genome Res. 2009, 19(2):327–335. 
+* Thanki AS, Ayling S, Herrero J, Davey RP: Aequatus: An open-source homology browser. bioRxiv 055632; doi: http://dx.doi.org/10.1101/055632
+
+# <a name="contacts"></a> Project contacts:
```
Comment (nsoranzo) [inline workflow/GeneSeqToFamily/readme.md]: Remove `<a name="contacts"></a>`

**Comment by anilthanki** [comment]
No we don't, but I had practice of putting GPL licence so did it, we can change it to MIT

**Comment by nsoranzo** [comment]
OK, can you remove the LICENCE file then? The rest should be ready for merge.

**Comment by anilthanki** [comment]
Done...

**Pull request merged**